plugins/copr.py

```diff
@@ -529,6 +562,13 @@ def _disable_repo(self, copr_username, copr_projectname):
             self.base.conf.write_raw_configfile(repo.repofile, multilib_repo_id,
                                                 self.base.conf.substitutions, {"enabled": "0"})
 
+        # disable runtime dependencies as well
+        for repo_id in repo.cfg.sections():
+            if repo_id.startswith("copr:"):
+                continue
```

Comment:
It is much nicer now, but now I see potential to remove the special-cased
`copr:...` and `copr:...:ml` repos.  Could you consolidate the code and disable
everything by this loop?